### PR TITLE
Feature/making pledges

### PR DESCRIPTION
**Pull request by DominikGuzei**
The primary intent of this PR was to make it possible to make pledges for open appeals.
While working on it i found the need to switch to another css framework, as zurb foundation does not deliver the style and quality i was looking for. During the switch to [materialize css](http://materializecss.com/) i realized that all other parts of the system also have to be updated to the new framework / style. Thus this PR became really big … sorry for that!
## Pledging

bildschirmfoto 2015-12-09 um 01 10 30

bildschirmfoto 2015-12-09 um 01 08 08
## Here are other improvements:
- Complete redesign / implementation of all parts using materialize css
- Beautiful open-appeals list
- More details in the `Donations.OpenAppealsProjection` (`location` and `organization`), so that we don't have to join that data on the client (which would not be easy as soon as we have open appeals from multiple orgs / locations -> those would have to be all sent down to the client too!)
## Screenshots

Open appeals list:
bildschirmfoto 2015-12-08 um 15 41 01

Redesigned Dashboard:
bildschirmfoto 2015-12-08 um 15 41 37

Redesigned Location Admin:
bildschirmfoto 2015-12-08 um 15 42 07

also the login / register have been updated of course – just checkout the branch.

### CODE REVIEW

**Comment by DominikGuzei** [comment]
@darko-mijic and @rhyslbw this PR is now ready for review and merge :wink:

File: packages/app/source/client/pages/index/landing-page.html [hunk 1]
```diff
@@ -2,23 +2,23 @@
 <div id="landing-page">
 
   <section class="donate">
-    <header>
+    <div class="headline">
       <h1>{{i18n "donate_things"}}</h1>
       <h2>{{i18n "voluntary_donors"}}</h2>
```
Comment (rhyslbw) [inline packages/app/source/client/pages/index/landing-page.html:7]: Did I make this translation :-) What is this meant to be communicating?

Comment (DominikGuzei) [inline packages/app/source/client/pages/index/landing-page.html:7]: haha, yeah those are yours :wink:

Comment (rhyslbw) [inline packages/app/source/client/pages/index/landing-page.html:7]: Ha, maybe we can just drop this. It makes no sense in English, how's the German?! lol

Comment (DominikGuzei) [inline packages/app/source/client/pages/index/landing-page.html:7]: Yeah we will refactor all of the translations at some point … but i think it "kind of makes sense" like that – those are the headlines of the landing page :wink:

Comment (rhyslbw) [inline packages/app/source/client/pages/index/landing-page.html:7]: "Open Appeals" is great, it's the subheading that can go. Will consider that for later

**Pull request merged**